src/python/pants/backend/python/lint/black/rules.py

```diff
@@ -54,7 +54,7 @@ class Setup:
 
 
 def generate_argv(source_files: SourceFiles, black: Black, *, check_only: bool) -> Tuple[str, ...]:
-    args = []
+    args = ["-q"]
```

Comment:
Is there not a long option, like `--quiet`? (On my phone, would've checked for myself otherwise)